content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,198 +4,139 @@ Start receiving tickets
+*Operators* are the users who act as agents and respond to live chat requests from customers. The
+user who created the live chat channel is added by default.
 
-To add additional users, navigate and click on the live chat channel from the :guilabel:`Website
-Live Chat Channels` dashboard, and on the :guilabel:`Operators` tab, click :guilabel:`ADD`.
+To add additional users, click on the :guilabel:`Operators` tab, click :guilabel:`Add`.
 
-Then, click the check box next to the users to be added, and click :guilabel:`SELECT`.
-:guilabel:`New` operators can be created and added to the list, as well, by filling out the
-:guilabel:`Create Operators` form and then clicking :guilabel:`SAVE & CLOSE` (or :guilabel:`SAVE &
-NEW` for multiplerecord creations).
+Click the checkbox next to the users to be added, and click :guilabel:`Select`. Click
+:guilabel:`New` to create new operators. Then, click :guilabel:`Save & Close`, or :guilabel:`SAVE &
+NEW` to add multiple new operators.
 
 As well, current operators can be edited or removed by clicking on their respective boxes in the
 :guilabel:`Operators` tab, and then adjusting their form values, or by using one of the form buttons
-located at the bottom of the form, such as :guilabel:`REMOVE`.
+located at the bottom of the form, such as :guilabel:`Remove`.
 
 .. tip::
-   Users can add themselves as an operator by clicking the :guilabel:`Join` button on a *Live Chat*
-   channel.
+   Users can add themselves as an operator by clicking the :guilabel:`Join Channel` button on a
+   *Live Chat* channel.
+
+   .. image:: receiving_tickets/receiving-tickets-join-live-chat.png
+      :align: center
+      :alt: View of a live chat channel Kanban card with the join button emphasized.
 
 Modify channel options
 **********************
 
 The :guilabel:`Options` tab contains the visual and text settings for the live chat window.
 
-Change the text in the :guilabel:`Text of the Button` field to update the greeting displayed in the
-text bubble when the live chat button appears on the website.
+- :guilabel:`Notification Text`: this field updates the greeting displayed in the text bubble when
+  the live chat button appears on the website.
 
-Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open the chat
-window. This message will appear as though it is sent by a live chat operator, and should be an
-invitation to continue the conversation.
+- :guilabel:`Welcome Message`: this field changes the first message a visitor receives when they
+  open the chat window. This message appears as though it is sent by a live chat operator, and
+  should be an invitation to continue the conversation.
 
-Edit the :guilabel:`Chat Input Placeholder` to change the text that appears in the box where
-visitors will type their replies.
+- :guilabel:`Chat Input Placeholder`: this field changes the text that appears in the box where
+  visitors type their replies.
 
```

Comment:
These seem miss ordered- they have two columns (livechat button / livechat window)

You go from left to right....